### DISCUSSION TITLE
Chapter 1 guide: "PHREU" doesn't translate to "pli"

### DISCUSSION THREAD

**Discussion by tomklino** [aerickt/lapwing-for-beginners#11] [Q&A] [answered]
2 day beginner question :)

I installed plover and the lapwing plugin, following the [chapter 1 guide](https://lapwing.aerick.ca/Chapter-01.html) to type the word "complicated" - when I type the stroke PHREU it outputs PHREU  into the document. According to the guide it should type "pli". I'm afraid I may have installed something wrong. What might I be missing?

**Comment by aerickt** [comment, marked as answer]
This table represents sounds, not spelled letters:
![image](https://github.com/user-attachments/assets/828bebb3-4461-4887-8de0-0e94bb7b0fb4)

When writing words with multiple strokes, each stroke represents a sound (not necessarily a group of letters). So sometimes individual strokes aren't defined by themselves, but entire outline is. `PHREU` isn't defined as anything, so you get the raw steno when you try to write it.

In your case, you wrote `KOPL/PHREU`, which resulted in "come PHREU". However, when you finish the outline by writing `KAEUT`, Plover will backspace the last 7 characters and finish the word. With these multistroke outlines, you just need to write the whole thing to get the desired output.

**Reply by tomklino** [reply]
Thanks! (<- I wrote this in steno)
